src/app/admin/components/user-card/user-card.component.scss

```
@@ -4,6 +4,30 @@
     box-shadow: 0px 0px 12px 1px rgba(0, 0, 0, 0.05);
     width: 185px;
     border-radius: 5px;
+    transition: transform .2s;
+}
+
+.card:hover {
+    transform: scale(1.1);
+    color: $bright-blue;
+} 
+
+.card:hover .card__middle {
+    opacity: 0.9;
+    box-shadow: 0px 0px 12px 1px rgba(0, 0, 0, 0.05);
```

`0px` is redundant and should be replaced by `0` as per the listing rules we don't have.
```suggestion
    box-shadow: 0 0 12px 1px rgba(0, 0, 0, 0.05);
```